extensions/cache/src/main/resources/benchmark-analysis.xml

```diff
@@ -56,12 +56,12 @@
       <l:basic-operations-test test-name="on-max-throughput"
                              duration="${test.long.duration:1m}" total-threads="#{ 10 * ${top.iteration} + 10 }">
          <statistics>
-            <default>
+            <basic>
```

Comment:
I wasn't comfortable with something called 'default' not being the default anymore.